src/hierarchy.rs

```diff
@@ -139,6 +145,35 @@ pub enum ScopeType {
     VhdlPackage,
 }
 
+impl fmt::Display for ScopeType {
```

Comment:
I don't think `wellen` should provide an implementation for ["user-facing output"](https://doc.rust-lang.org/std/fmt/trait.Display.html) since it is a library. A user like Surfer can implement `Display` for `ScopeType` if needed.